code/game/objects/structures/false_walls.dm

```diff
@@ -11,7 +11,7 @@
 * around us, then checks the difference.
 */
 /proc/getOPressureDifferential(var/turf/loc)
-	var/minp=16777216;
+	var/minp=16777216; //What is even the significance of this number?
```

Comment:
According to my research, that number (2^24) is the highest value that can be stored in a 32-bit float with integer precision
So presumably pressure was SUPPOSED to be limited to that, but never actually was